tnz/zti.py

```diff
@@ -29,6 +29,7 @@
     ZTI_AUTOSIZE
     ZTI_SECLEVEL (see tnz.py)
     ZTI_TITLE
+    ZTI_MACROS_DIR (~/.zti-macros.d is default)
```

Comment:
I haven't tried yet... but does this cause some _strangeness_ on windows because it's longer than 8 characters? I wonder if a shorter name might make things more pleasant.